crates/amaru-kernel/src/network.rs

```diff
@@ -192,6 +194,46 @@ impl From<NetworkName> for &EraHistory {
     }
 }
 
+static PREPROD_GLOBAL_PARAMETERS: LazyLock<GlobalParameters> = LazyLock::new(|| {
+    // https://cips.cardano.org/cip/CIP-9
+    let consensus_security_param = 2160;
+    let active_slot_coeff_inverse = 20;
+    let shelley_epoch_length_scale_factor = 10;
+    let shelley_epoch_length =
+        active_slot_coeff_inverse * shelley_epoch_length_scale_factor * consensus_security_param;
+    let byron_epoch_length_scale_factor = 10;
+    let byron_epoch_length = byron_epoch_length_scale_factor * consensus_security_param;
+    let shelley_transition_epoch = 4;
+    GlobalParameters {
+        consensus_security_param,
+        shelley_epoch_length_scale_factor,
+        active_slot_coeff_inverse,
+        byron_epoch_length_scale_factor,
+        shelley_transition_epoch,
+        max_lovelace_supply: 45_000_000_000_000_000,
+        slots_per_kes_period: 129_600,
+        max_kes_evolution: 62,
+        shelley_epoch_length,
+        stability_window: active_slot_coeff_inverse * consensus_security_param * 2,
+        byron_epoch_length,
```

Comment:
Note that the "byron_epoch_length", "byron_epoch_length_scale_factor" and "shelley_transition_epoch" should no longer be needed now that we started using the Era history for slot arithmetic. They should probably be removed from the globals to avoid shooting ourselves in the foot later. 

Which would also mean we can then rename "shelley_epoch_length" to simply "epoch_length".